elastic/kibana
# [Security Solution][Exceptions] - Fixes exception builder bug that includes matches operator

## Summary

Addresses https://github.com/elastic/kibana/issues/136224

### Testing
- Create a rule
- Go to rule details and add an exception
- Select a field and check that the "matches" operator is not available

### Checklist

    -  Ticket - https://github.com/elastic/security-docs/issues/2199
